src/d3a/models/myco_matcher/pay_as_bid.py

```diff
@@ -0,0 +1,45 @@
+from d3a.constants import FLOATING_POINT_TOLERANCE
+
+
+class PayAsBidMatch:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def sorting(obj, reverse_order=False):
+        if reverse_order:
+            # Sorted bids in descending order
+            return list(reversed(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate)))
+
+        else:
+            # Sorted bids in ascending order
+            return list(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate))
+
+    def calculate_match_recommend(self, bids, offers):
```

Comment:
I think this might be a typo for `calculate_match_recommendation`. Is this correct?